requirements/edx/custom.txt

```diff
@@ -23,11 +23,11 @@
 -e git+https://github.com/mckinseyacademy/xblock-diagnosticfeedback.git@v0.2.4#egg=xblock-diagnostic-feedback==0.2.4
 -e git+https://github.com/open-craft/xblock-group-project-v2.git@0.4.9#egg=xblock-group-project-v2==0.4.9
 -e git+https://github.com/open-craft/xblock-virtualreality.git@v0.1.1#egg=xblock-virtualreality==0.1.1
-git+https://github.com/edx-solutions/api-integration.git@v2.0.10#egg=api-integration==2.0.10
+git+https://github.com/edx-solutions/api-integration.git@v2.0.10#egg=api-integration==2.1.4
```

Comment:
@Agrendalath So we already merged this but I noticed it brought up an issue with the latest api-integration. We need to bump the `v2.0.10` to `v2.1.4` as well here. I think tests are failing with that change - can you check it out?

Reply:
Right, this one and also discussions and gradebooks. I pushed the commit to #1224.